src/test/test-chase.c

```diff
@@ -285,18 +288,18 @@ TEST(chase) {
         assert_se(r == 0);
```

Comment:
Please change to `ASSERT_OK_ERRNO()` for `chdir()`

Also, ASSERT_OK_ZERO(chase(...))